styles.css

```diff
@@ -0,0 +1,555 @@
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.conto {
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 52px;
+  align-items: center;
+  color: var(--Primary);
+}
+
+.details {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+  margin-top: 5%;
+  width: 327px;
+}
+
+.details input {
+  font-size: 24px;
+  border-color: #fff;
+}
+
+#name {
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+#email {
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+#message {
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #60c095;
+}
+
+.bt-sec {
+  margin-right: 30%;
+  margin-top: 5%;
+}
+
+.bt {
+  background: #36b37f;
+  border: none;
+  padding: 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #fff;
+  width: 129px;
+  height: 48px;
+}
+
+.bt:hover {
+  background-color: #008552;
+  cursor: pointer;
+}
+
+.foot {
+  justify-content: center;
+  display: flex;
+  align-items: center;
+}
+
+.social-2 {
+  display: flex;
+  margin-top: 2%;
+  gap: 10px;
+}
+
+.mail {
+  margin-top: 2%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@media screen and (max-width: 786px) {
+  .header-content {
+    margin: 5%;
+  }
+
+  .logo {
+    visibility: visible;
+    color: #28352f;
+  }
+
+  .burger {
+    display: block;
+    flex-direction: row;
+    font-size: 1.5rem;
+    margin-right: 10%;
+  }
+
+  .nav-list {
+    display: none;
+  }
+
+  .hero {
+    background-image: url(assest/img/bgimg/header-illsutration-mobile.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: auto;
+    height: 100vh;
+    position: relative;
+  }
+
+  h1 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    color: var(--Primary);
+  }
+
+  h2 {
+    color: var(--Secondary);
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    width: 327px;
+  }
+
+  .next {
+    margin-top: 5%;
+    display: flex;
+  }
+
+  .head {
+    width: 327px;
+    margin-left: 8%;
+  }
+
+  .cards {
+    grid-template-columns: auto;
+    padding-left: 2%;
+  }
+
+  .contant-1 {
+    left: 25%;
+    background-image: url(assest/img/aboutme/mob-\ about\ me-\ mobile.png), url(assest/img/aboutme/mob-2-\ about\ me-mobile.png);
+    background-size: auto;
+    background-repeat: no-repeat;
+    background-position: left bottom, right top;
+    width: 50%;
+    height: 50vh;
+  }
```

Comment:
- [ ] Just as the previous reviewer discussed, I think you should incest more time to get the background tight. It isn't looking too good at the moment.
![image](https://user-images.githubusercontent.com/99715640/219560635-b648f47c-5312-4e6d-a259-f17f4eff49f2.png)